__tests__/tests/utilities/migration-helper.js

```diff
@@ -0,0 +1,44 @@
+import {getRepoIdFromUrl, toOldIssueFormat, toOldUserFormat} from 'utils'
+
+describe('Migration Helper', () => {
+  it('should return a repo id in {username}/{reponame} format', () => {
+    const result = getRepoIdFromUrl('https://api.github.com/repos/octocat/Hello-World');
+    expect(result).toBe('octocat/Hello-World');
+  });
+
+  it('should convert a user object to old format', () => {
+    const user = {};
+
+    const result = toOldUserFormat(user);
+    const resultKeys = Object.keys(result);
+    expect(result.avatar_url).toBe(user.avatarUrl);
+    expect(Object.keys(user).reduce((accum, key) => accum && resultKeys.includes(key), true)).toBe(true);
+  });
+
+  it('should convert an issue object to old format', () => {
+    const issue = {
+      author: {},
+      comments: {
+        totalCount: 7,
+      },
+      closedAt: '',
+      createdAt: '',
+      number: 42,
+      state: 'StAte',
+    };
+    const repoId = 'git-point';
+
```

Comment:
Why is there a space? Let's add one a space between the variable declarations and the `expect` calls, and stick with it in all `it` blocks, what do you think?

```js
const a = 'something';
const b = 'something else';

expect(...);
expect(...);
```